spanner/integration_test.go

```diff
@@ -4328,6 +4329,85 @@ func TestIntegration_GFE_Latency(t *testing.T) {
 	DisableGfeLatencyAndHeaderMissingCountViews()
 }
 
+// TestIntegration_DropDatabaseProtection tests the drop database protection feature
```

Comment:
Can we add an integration test to check if a non-null value is populated for the enable_drop_protection field in ListDatabases call as well? This field will be back-populated for old databases as well.

Reply:
As `enable_drop_protection` field is a boolean field, in golang this automatically defaults to `false` and cannot have a `nil` value in it. Hence instead of testing `enable_drop_protection` field, I have added a test to verify `ListDatabases` returns `non-null` databases which by default indicates that `enable_drop_protection` field is set to either `true`/`false`.